### PR TITLE
Official US President Update

### PR DESCRIPTION
**Pull request by omarhachach**
Since he has won the presidential race, I thought a few changes to the front page should be made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 Make Python great again
 
 ## Mission
-TrumpScript is a language based upon the illustrious Donald Trump. As the undeniably best presidential candidate in the 2016 language, we found that the current field of programming languages does not include any that Trump's glorious golden combover would approve of.
+TrumpScript is a language based upon the illustrious Donald Trump. As the undeniably best president ever, we found that the current field of programming languages does not include any that Trump's glorious golden combover would approve of.
```
Comment (jibeiz) [inline README.md:5]: isn't it a bit too early to say that?

Comment (omarhachach) [reply README.md:5]: Nope. Unless he would get assassinated, he is president.
You can wait untill he actually becomes president though.

Comment (gunsworth) [reply README.md:5]: President-elect would make sense since that's the current verbage being used to describe him by most news outlets.

Comment (EdgeCaseBerg) [reply README.md:5]: why not list everything? "the undeniably best presidential candidate, president-elect, and president ever" 
A bigly list for a bigly prez

Comment (omarhachach) [reply README.md:5]: @gebellamy yeah.
@EdgeCaseBerg good idea.

**Pull request closed without merging**